### ISSUE TITLE
It's Alive!

### ISSUE THREAD

**Issue by samvel2804** [HellscreamWoW/Tracker#3661]
This quest "It's Alive!" is a following quest from "Heart of the Watcher" but its offered beffor you find a golems hearth or rather beffor "Heart of the Watcher" is completed:
quest to fix: http://www.wowhead.com/quest=26257/its-alive
quest that must be done first: http://www.wowhead.com/quest=26252/heart-of-the-watcher
kittens: http://www.southernhillsanimalhospital.com/sites/site-1450/images/kittens.jpg

**Comment by carminejackal** [comment]
https://github.com/HellscreamWoW/Tracker/issues/406 https://github.com/HellscreamWoW/Tracker/issues/1018

**Comment by GMTyrial** [comment]
Duplicate of https://github.com/HellscreamWoW/Tracker/issues/1018
